sbpy/activity/gas/tests/test_prodrate.py

```diff
@@ -1,3 +1,15 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from .. import Haser, photo_timescale
+from ....data import Ephem, Phys
+from .. import (LTE, NonLTE, einstein_coeff, intensity_conversion, beta_factor,
```

Comment:
`NonLTE` will be an only remote test (not needed here)?

Reply:
You're correct, thanks!